source/dem/dem.cc

```diff
@@ -1064,9 +1064,12 @@ DEMSolver<dim>::write_output_results()
       particles_force_chains_object =
         set_force_chains_contact_force_model(parameters);
       particles_force_chains_object->calculate_force_chains(contact_manager);
-      particles_force_chains_object->write_force_chains(this->mpi_communicator,
+      particles_force_chains_object->write_force_chains(particles_pvdhandler,
+                                                        this->mpi_communicator,
                                                         folder,
-                                                        iter);
+                                                        group_files,
+                                                        iter,
+                                                        time);
```

Comment:
eventually this could be a tiny struct, can you open an issue for this? We could have everywhere a small struct called io_information or something like that. Not in this PR but in a future one.